lib/iris/tests/test_grib_save.py

```diff
@@ -34,7 +34,7 @@
 
 if tests.GRIB_AVAILABLE:
     import gribapi
```

Comment:
@marqh I don't understand why this import is still here.  Is this not being replaced?